text/faq.yml

```diff
@@ -25,13 +24,13 @@ faq:
         The data used in DGIdb is all open access and where possible made available as raw data dumps in the 
         <a href="/downloads">downloads</a> section.
 
-    - question: How can I learn more about how to use DGIdb?
-      anchor: learn
-      answer: > 
-        You can try reading: the <a href="/getting_started">'Getting Started'</a> section under the help menu,
-         the <a href="https://doi.org/10.1093/nar/gkx1143">manuscript</a>, and this FAQ.
-        Some sections also contain a 'Tour' that can be activated by pressing the 'Show Tour' button in the top left.
-        If all else fails, please <a href="mailto:help@dgidb.org">email us</a> with your questions.
+    #- question: How can I learn more about how to use DGIdb?
+    #  anchor: learn
```

Comment:
Is this intended to be commented rather than removed?

Reply:
I figured we might want to add this again if we ever add another set of tutorials.